### PR TITLE
CHIT leaderboard backend

### PR DESCRIPTION
**Pull request by megrogan**

### CODE REVIEW

**Review by hpeebles — DISMISSED** [review]

File: backend/libraries/types/src/chit.rs [hunk 1]
```diff
@@ -15,3 +15,9 @@ pub enum ChitEarnedReason {
     DailyClaim,
     Achievement(String),
 }
+
+#[derive(CandidType, Serialize, Deserialize, Clone, Debug)]
+pub struct ChitUserBalance {
+    pub user_id: UserId,
+    pub balance: i32,
```
Comment (hpeebles) [inline backend/libraries/types/src/chit.rs:22]: Maybe this should be a u32? And we exclude negatives.

Comment (hpeebles) [reply backend/libraries/types/src/chit.rs:22]: Otherwise if the first user in a season gets a deduction, they'll still appear top.

**Comment by megrogan** [comment]
Not ready yet... still need to remove a user if there balance has gone negative

**Comment by megrogan** [comment]
Latest commit means the leaderboard could have < 50 users even though there are at last 50 postive balances but this is very unlikely in practise

**Review by hpeebles — APPROVED** [review]

**Pull request merged**